arbitrum-docs/node-running/how-tos/running-an-orbit-node.mdx

```diff
@@ -0,0 +1,79 @@
+
+
+### Minimum hardware configuration
+
+- Followings specify the minimum hardware configuration required to setup a Nitro full node (not archival):
+  - RAM: 8-16 GB
+  - CPU: 2-4 core CPU (For AWS: t3 xLarge)
+  - Storage: It depends on the orbit chain and its traffic during the time. 
+
+### Prerequisites
+
+- Latest Docker Image: <code>@latestNitroNodeImage@</code>
+
+### Required parameters
+
+- Parent chain RPC URL
+  - Use the parameter `--parent-chain.connection.url=<Parent chain RPC URL>`
+  - It must provide a standard RPC endpoint for an EVM node, whether self-hosted or obtained from a node service provider
+    - Would be set to appropriate Arbitrum One, Arbitrum Nova, or Arbitrum Sepolia RPC URL
+  - Note: [Arbitrum RPC endpoints](/node-running/node-providers.mdx/#rpc-endpoints) RPCs rate limit connections, So you will need to run a local node for the parent chain or rely on third party RPC providers
+- L3 orbit chain info and chain name
+  - Providing the flag --chain.info-json=<Orbit Chain's chain info> is mandatory. The chain info json string contains vital information about the Orbit chain, which is necessary for the node to commence operations. An example of the chain info json string is available in the subsequent sections.
+  - `--chain.name=<My Arbitrum L3 Chain>` must be provided, and it must match the chain name that is used in `--chain.info-json` content.
+  - For an Anytrust chain, it is essential to provide `--node.data-availability.enable`.
+
+### Important ports
+
+- RPC: `8547`
+- Sequencer Feed: `9642`
+- WebSocket: `8548`
+  - WS port `8548` needs extra args to be opened. Please use these flags:
+    - --ws.port=8548
+    - --ws.addr=0.0.0.0
+    - --ws.origins=\*
+
+### Putting it all together
+
+- When running docker image, an external volume should be mounted to persist the database across restarts. The mount point inside the docker image should be `/home/user/.arbitrum`
+- Here is an example of how to run orbit-node:
+
+  - Note that is important that `/some/local/dir/arbitrum` already exists, otherwise the directory might be created with `root` as owner, and the docker container won't be able to write to it
+
+  ```shell
+  docker run --rm -it  -v /some/local/dir/arbitrum:/home/user/.arbitrum -p 0.0.0.0:8547:8547 -p 0.0.0.0:8548:8548 @latestNitroNodeImage@ --parent-chain.connection.url=<Parent chain RPC URL> --chain.id=<L3ChainId> --chain.name=<My Arbitrum L3 Chain> --http.api=net,web3,eth,debug --http.corsdomain=* --http.addr=0.0.0.0 --http.vhosts=* --chain.info-json=<Orbit Chain's chain info>
+ 
+  ```
```

Comment:
How about we also introduce how to edit and use the `nodeconfig.json` file here? Because we introduce to use that at `orbit-setu-script`